src/net.h

```diff
@@ -677,6 +677,7 @@ class CNode
 
 
     uint64_t nLocalHostNonce;
+    // What local services you advertised on connection to this node
```

Comment:
Yes, different services may be advertised to different peers, based on, say, whitelisting.